Client/Frontend/Theme/LegacyThemeManager/ThemedTableSectionHeaderFooterView.swift

```diff
@@ -63,12 +63,20 @@ class ThemedTableSectionHeaderFooterView: UITableViewHeaderFooterView, ReusableC
         fatalError("init(coder:) has not been implemented")
     }
 
+    // TODO: FXIOS-4884 - NotificationThemeable applyTheme
+    // to remove in favor of applyTheme(theme: Theme) and updateThemeApplicableSubviews
     func applyTheme() {
         bordersHelper.applyTheme()
         contentView.backgroundColor = UIColor.theme.tableView.headerBackground
         titleLabel.textColor = UIColor.theme.tableView.headerTextLight
     }
```

Comment:
Is this because it's used in more locations than the ones updated by this PR?

Reply:
Yeah exactly, it entailed too many changes to put a stop there. This will be fixed in the next PR